src/main/java/com/gregtechceu/gtceu/data/lang/LangHandler.java

```diff
@@ -1128,7 +1130,7 @@ public static void init(RegistrateLangProvider provider) {
         provider.add("gtceu.multiblock.universal.distinct.info",
                 "If enabled, each Item Input Bus will be treated as fully distinct from each other for recipe lookup. Useful for things like Programmed Circuits, Extruder Shapes, etc.");
         provider.add("gtceu.multiblock.parallel", "Performing up to %d Recipes in Parallel");
-        provider.add("gtceu.multiblock.parallel.exact", "Performing %d Recipes in Parallel");
+        provider.add("gtceu.multiblock.parallel.exact", "- %dx Parallels");
```

Comment:
```suggestion
        provider.add("gtceu.multiblock.parallel.exact", "- %dx from Other Modifiers");
```
Maybe could just be `from Parallels`, but this category includes not just the parallel hatch but any other modifiers that might apply, like steam multiblocks or multismelter - imo it should still be 'from Something' to match the format of the other two